Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiRequestBus.h

```diff
@@ -104,6 +104,10 @@ namespace AzToolsFramework::ViewportUi
         virtual void SetTextFieldVisible(TextFieldId textFieldId, bool visible) = 0;
         //! Create the highlight border with optional back button to exit the given editor mode.
         virtual void CreateViewportBorder(const AZStd::string& borderTitle, AZStd::optional<ViewportUiBackButtonCallback> backButtonCallback) = 0;
+        //! Check if the viewport border is visible.
```

Comment:
update: Better to use phrasing to say what this does, not what it's doing. So this would be `Returns if the viewport border is visible.`